### PR TITLE
Add a flag to force Babylon to use OpenGL convention for UVs

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

File: src/Meshes/Builders/capsuleBuilder.ts [hunk 1]
```diff
@@ -131,7 +132,7 @@ export function CreateCapsuleVertexData(
             normal.set(sinTheta, slope, cosTheta).normalize();
             normals.push(normal.x, normal.y, normal.z);
             // uv
-            uvs.push(u, 1 - v / vl);
+            uvs.push(u, v / vl ? v / vl : 1 - v / vl);
```
Comment (Popov72) [inline src/Meshes/Builders/capsuleBuilder.ts:135]: Should be `CompatibilityOptions.UseOpenGLOrientationForUV ? v / vl : 1 - v / vl`

File: src/Meshes/geometry.ts [hunk 2]
```diff
@@ -1208,31 +1209,61 @@ export class Geometry implements IGetSetVerticesData {
 
             if (binaryInfo.uvsAttrDesc && binaryInfo.uvsAttrDesc.count > 0) {
                 var uvsData = new Float32Array(parsedGeometry, binaryInfo.uvsAttrDesc.offset, binaryInfo.uvsAttrDesc.count);
+                if (CompatibilityOptions.UseOpenGLOrientationForUV) {
+                    for (var index = 1; index < uvsData.length; index += 2) {
+                        uvsData[index] *= -1;
```
Comment (Popov72) [inline src/Meshes/geometry.ts:1214]: Why not `1-uvsData[index]`? I think it's not the same thing than `*= -1` if the address mode is CLAMP.

For eg, if v=0.3, the current code will compute -0.3 and in CLAMP mode you will get 0. With `1-v` you get 0.7 and still get 0.7 in CLAMP mode.

Same comment for all the loops below.

File: src/Meshes/Builders/capsuleBuilder.ts (hunk 1 above)
Comment (deltakosh) [reply src/Meshes/Builders/capsuleBuilder.ts:135]: lol yes!! bad copy paste

**Review by RaananW — APPROVED** [review]

**Comment by RaananW** [comment]
There is a case issue (first letter should be a capital C) with the file you have added. Not noticeable in windows, failing in Unix

**Pull request merged**